tla-import/src/main/scala/at/forsyte/apalache/tla/imp/Context.scala

```diff
@@ -100,7 +133,7 @@ object Context {
    * Create a new context, i.e., use Context().
    */
   def apply(): Context = {
-    new ContextImpl(List(), List(), Map())
+    new ContextImpl(List(), false, List(), Map())
```

Comment:
Is there a way, or need, to ever pass `true` here w/o manually constructing `ContextImpl`? Not sure I like mutating after construction, if it's not necessary.
In general, it would be nice to have a line comment, explaining the default value.

Reply:
Well, yes, they are related. However, I am trying to keep the imported IR close to the source. In case of LOCAL, we have no other choice but to introduce unique namespaces.